.circleci/config.yml

```diff
@@ -308,6 +341,10 @@ jobs:
     executor: bionic
     steps:
       - test-chrome
+  test-sockets-chrome:
```

Comment:
This may as well use upstream (build-upstream instead of build, etc.), so that it doesn't get removed when we remove fastcomp. Also, upstream tests run faster.